nodes/signal_shutdown.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# Simple script for shutting down the raspberry Pi at the press of a button.
+# by Inderpreet Singh
+# Adapted by Jake Rye and Bob Davis
+
+import RPi.GPIO as GPIO
```

Comment:
Unfortunately this wasn't as easy as we might have hoped. I found out the permissions error is a timing issue because the gpio pins need to be setup before sending commands to them. I submitted an issue for this on the python-periphery package: [here](https://github.com/vsergeev/python-periphery/issues/10)

The other problem is the python-periphery package doesn't have a pull_up feature when setting up a pin to monitor. The RPi.GPIO library allows the pin to be pulled high and then watch when it goes low. The python-periphery package drops the pin low when set up as an input. I don't see how to get around this issue at this time.

I'd recommend we land this change using the RPi.GPIO library and identify another library or fix if we decide to move to different hardware. Does that sound reasonable?